specification/appconfiguration/AppConfiguration/tspconfig.yaml

```diff
@@ -26,6 +26,7 @@ options:
     model-namespace: false
     flavor: azure
   "@azure-typespec/http-client-csharp":
+    package-name: Azure.Data.AppConfiguration
```

Comment:
I think this the package-name is defaulted to the namespace, so not strictly necessary.